service/src/main/kotlin/io/provenance/api/frameworks/provenance/extensions/ProvenanceExtensions.kt

```diff
@@ -9,16 +9,19 @@ import cosmos.tx.v1beta1.ServiceOuterClass.BroadcastTxResponse
 import cosmos.tx.v1beta1.TxOuterClass
 import io.provenance.api.frameworks.provenance.BatchTx
 import io.provenance.api.frameworks.provenance.SingleTx
+import io.provenance.api.models.p8e.AudienceKeyPair
 import io.provenance.client.grpc.PbClient
 import io.provenance.client.protobuf.extensions.getBaseAccount
 import io.provenance.scope.contract.proto.Contracts
+import io.provenance.scope.encryption.util.getAddress
+import io.provenance.scope.encryption.util.toJavaPublicKey
 import java.util.concurrent.TimeUnit
 
 fun BroadcastTxResponse.isError() = txResponse.isError()
 
 fun TxResponse.isError() = code > 0
 
-fun TxResponse.isSuccess() = !isError() && height > 0
+// fun TxResponse.isSuccess() = !isError() && height > 0
```

Comment:
I commented this out because it was one of a few unused functions in this file and the linter was giving the error TooManyFunctions. Can remove or address it differently if desired.

Reply:
Uncommented this out and added the suppression, doesn't bother me 🤷🏽